test/user/profile.test.js

```diff
@@ -0,0 +1,19 @@
+import request from 'supertest';
+import chai from 'chai';
+import app from '../../server';
+
+const { expect } = chai;
+
+describe('GET /api/profile', () => {
+  it('Should return 200 on get profile call', (done) => {
+    request(app)
+      .get('/api/usergames')
+      .expect(200)
+      .end((req, res) => {
+        expect(res.body).to.have.property('data');
+        expect(res.body).to.have.property('code');
+        expect(res.status).to.equal(200);
+        done();
+      });
+  });
+});
```

Comment:
Your describe block suggests you that this test suite is for this endpoint `/api/profile` However, you seem to have made a `get` request to `/api/usergames`. Please verify what endpoint this test is valid for.

Reply:
The language used here `Should return 200 on get profile call` does not see to communicate what you are testing for. Once you respond to the feedback about you should have more clarity on what this should have been framed as.

Reply:
I believe you should have been able to provide more test specs that prove you have considered other edge cases. E.g. the user has not played a single game, what is returned in that case?